### PR TITLE
sign-up

### PR DESCRIPTION
**Pull request by mariangz**
The sign up form must content:

- [x] name
- [x] surname
- [x] email
- [x] password and confirm password
- [x] use date
- [x] use checkbox
- [x] use radio
- [x] use datalist

### CODE REVIEW

**Comment by AlbaHerrerias** [comment]
I'm guessing this is not yet ready as you haven't put me as a reviewer 🤔  Not sure if you're familiar with it, but in case you're not, you can mark this PR as a draft to explicitly say it's not ready

**Comment by mariangz** [comment]
Screenshot 2022-05-04 at 10 10 27

When you hover over that icon ('?'), it will give more information about what types of characters your password needs

File: src/FormComponents/Password/Password.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <span
+          className='eye-toggle'
+          onClick={() => setShowPassword((prevValue) => !prevValue)}
+        >
+          {icon}
+        </span>
+      </div>
+
+      {meta.touched && meta.error && (
+        <p className='error'>Password must include at least 8 characters.</p>
```
Comment (AlbaHerrerias) [inline src/FormComponents/Password/Password.js:50]: As we talk, change the text to `{meta.error}`

**Review by AlbaHerrerias — APPROVED** [review]
✨  great job

**Pull request merged**